src/tools/inner_node.rs

```diff
@@ -100,3 +108,22 @@ impl InnerNode {
         self.node.shut_down().await
     }
 }
+
+fn decode_to_vec(base58str: &str, size: usize) -> bs58::decode::Result<Vec<u8>> {
+    let mut bytes = bs58::decode(base58str)
+        .with_alphabet(bs58::Alphabet::RIPPLE)
+        .into_vec()?;
+    bytes.remove(0);
+    bytes.truncate(size);
+    Ok(bytes)
+}
+
+fn decode_keys() -> Result<(SecretKey, PublicKey), secp256k1::Error> {
+    let bytes = decode_to_vec(SYNTHETIC_NODE_PRIVATE_KEY, SECRET_KEY_SIZE)
+        .expect("Unable to decode the private key.");
+    let private_key = SecretKey::from_slice(bytes.as_slice())?;
+    let bytes = decode_to_vec(SYNTHETIC_NODE_PUBLIC_KEY, PUBLIC_KEY_SIZE)
+        .expect("Unable to decode the public key.");
+    let public_key = PublicKey::from_slice(bytes.as_slice())?;
+    Ok((private_key, public_key))
```

Comment:
Maybe we could add a few empty lines between different logical line blocks to make it a little bit clear 🤔

Reply:
Done